packages/react/src/accordion-item/AccordionItem.tsx

```diff
@@ -1,16 +1,22 @@
 import * as RadixAccordion from "@radix-ui/react-accordion";
 import { forwardRef } from "react";
 
-import { Box, type BoxProps } from "../box";
+import { type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { extractSprinkles } from "../sprinkles";
 
 type AccordionItemProps = BoxProps<typeof RadixAccordion.Item>;
 
 export const AccordionItem = forwardRef<HTMLDivElement, AccordionItemProps>(
   ({ children, value, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
     return (
-      <Box asChild borderT="1" ref={ref} {...props}>
-        <RadixAccordion.Item value={value}>{children}</RadixAccordion.Item>
-      </Box>
+      <Flex asChild gap="4" ref={ref} {...sprinkleProps}>
```

Comment:
we cannot use Flex for the accordion item because accordions can collapse and expand. and the way this toggling works is using JS we animate the height of accordion content from 0 to it's intrinsic height when opening and then back to 0 when closing.

but if you use a flexbox and define a gap between the trigger and the content - then you'll notice "jumps" when the content is displayed with a 0 height because at that point flexbox will introduce the gap in between the trigger and the content.

you can see this clearly when you remove text from the content and then toggle it (i've also removed padding from the content to illustrate the jump further)

https://github.com/user-attachments/assets/51453b3b-a7c3-450e-8533-3923e88a164f